kibana/connector.py

```diff
@@ -113,18 +113,12 @@ def delete(self, uri, params=None, error=True, **kwargs):
 
     def login(self, kibana_username, kibana_password):
         """Authenticate to Kibana using the API to update our cookies."""
-        payload = {'username': kibana_username, 'password': kibana_password}
+        payload = {'params': {'username': kibana_username, 'password': kibana_password},
+                   'currentURL': '', 'providerType': 'basic',
+                   'providerName': 'cloud-basic' if self.cloud_id else 'basic'}
```

Comment:
small nitpick for readability
```suggestion
        payload = {
            'params': {
                'username': kibana_username,
                'password': kibana_password
            },
            'currentURL': '', 'providerType': 'basic',
            'providerName': 'cloud-basic' if self.cloud_id else 'basic'
        }
```